### ISSUE TITLE
Feature Request: Sidebar items for Main Panel

### ISSUE THREAD

**Issue by ItsCharged** [matt8707/ha-fusion#510]
Read Title. This includes Almosen all Things such as clock, bar, Template Card, Website Card etc.

**Comment by Quaddel** [comment]
Yes, please! That would be so cool. I could show all my cameras on one page ❤️
